dxr/query.py

```diff
@@ -78,23 +108,31 @@ def _line_query_results(self, filters, results, path_highlighters):
     def _file_query_results(self, results, path_highlighters):
         """Return an iterable of results of a FILE-domain query."""
         for file in results:
-            yield (icon(file['path'][0]),
-                   highlight(file['path'][0],
+            yield ('folder' if file['is_folder'] else icon(file['path'][0]),
+                   highlight_path(file['path'][0],
                              chain.from_iterable(
                                  h(file) for h in path_highlighters)),
                    [],
                    file.get('is_binary', False))
 
-    def results(self, offset=0, limit=100):
-        """Return a count of search results and, as an iterable, the results
-        themselves::
+    def instantiate_filters(self):
+        """Instantiate applicable filters, yielding a list of lists, each inner list
```

Comment:
Here and in several other places, you wrap a docstring to a funny width. If we wrap them, it should be to 79 columns.